bbb-common-web/src/main/java/org/bigbluebutton/api/ApiParams.java

```diff
@@ -77,6 +77,7 @@ public class ApiParams {
     public static final String PRESENTATION_UPLOAD_EXTERNAL_DESCRIPTION = "presentationUploadExternalDescription";
     public static final String PRESENTATION_UPLOAD_EXTERNAL_URL = "presentationUploadExternalUrl";
 
+    public static final String BREAKOUT_ROOMS_INCLUDE_MODS  = "breakoutRoomsIncludeMods";

```

Comment:
I think this information will not be relevant within the BreakoutRooms.
All changes in bbb-web can be removed!